### DISCUSSION TITLE
unable to find version `master`

### DISCUSSION THREAD

**Discussion by OscarKjell** [community/community#37814] [Actions] [answered]
A few days ago I started to get an error on github actions that had been working fine before: 

`Unable to resolve action `r-lib/actions@master`, unable to find version `master` `

After checking the r-lib/actions page (https://github.com/r-lib/actions/tree/v2-branch), I've tried changing it to: `r-lib/actions@v2` and `r-lib/actions@v2-branch`. but I still get the same error. 

The workflow file that I use can be found here:  https://github.com/OscarKjell/text/actions/runs/3356674880/workflow

Any help is much appreciated. 
Thanks in advance. 
Oscar

**Comment by airtower-luna** [comment, marked as answer]
You changed only one use of `r-lib/actions`, there's one left that references `master`:  https://github.com/OscarKjell/text/actions/runs/3356674880/workflow#L40

**Reply by OscarKjell** [reply]
thanks a lot!
